package.json

```diff
@@ -6,74 +6,52 @@
   "license": "AGPL-3.0",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve resources/js/src/index.ts --https --key private.key --cert private.pem --disable-host-check --hot",
-    "dev": "vue-cli-service build resources/js/src/index.ts --mode=development --modern",
-    "lint": "vue-cli-service lint resources/js/src resources/js/types",
-    "build": "vue-cli-service build resources/js/src/index.ts --mode=production",
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "lint": "eslint --ext .js --ext .vue --ext .ts resources/js",
+    "lint-fix": "eslint --ext .js --ext .vue --ext .ts resources/js --fix",
     "test": "jest --coverageDirectory=jest"
   },
   "files": [
     "resources/js/types"
   ],
   "dependencies": {
     "@tanem/svg-injector": "^10.1.19",
+    "@vue/cli-plugin-eslint": "^5.0.8",
     "@vue/composition-api": "^1.0.0-rc.5",
-    "vue": "^2.6.11",
+    "vue": "^2.6.14",
     "vue-class-component": "^7.2.3",
-    "vue-property-decorator": "^9.1.2"
+    "vue-property-decorator": "^9.1.2",
+    "vue-template-compiler": "2.6.14"
   },
   "devDependencies": {
-    "@babel/core": "^7.17.5",
-    "@babel/plugin-proposal-object-rest-spread": "^7.17.3",
-    "@babel/plugin-transform-typescript": "^7.16.8",
-    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-env": "^7.20.2",
     "@types/jest": "^27.4.1",
     "@types/jquery": "^3.5.14",
     "@types/jqueryui": "^1.12.16",
     "@types/node": "^17.0.21",
     "@typescript-eslint/eslint-plugin": "^5.15.0",
     "@typescript-eslint/parser": "^5.15.0",
-    "@vue/cli-plugin-babel": "^5.0.1",
-    "@vue/cli-plugin-eslint": "^5.0.1",
-    "@vue/cli-plugin-typescript": "^5.0.1",
-    "@vue/cli-plugin-unit-jest": "^5.0.1",
-    "@vue/cli-service": "^5.0.1",
+    "@vitejs/plugin-vue": "^1.6.1",
     "@vue/eslint-config-typescript": "^10.0.0",
     "@vue/test-utils": "^1.3.0",
-    "@vue/vue2-jest": "^27.0.0-alpha.4",
+    "@vue/vue2-jest": "^27.0.0",
     "autoprefixer": "^9.0.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-jest": "^27.5.1",
-    "babel-loader": "^8.2.3",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-es2015": "^6.24.1",
-    "css-loader": "^5.0.2",
-    "eslint": "^8.10.0",
-    "eslint-plugin-vue": "^8.5.0",
+    "eslint": "^8.35.0",
+    "eslint-plugin-vue": "^8.7.1",
     "jest": "^27.5.1",
     "jquery": "^3.6.0",
```

Comment:
Please check if `jquery` is still used. I don't think that's the case.

Reply:
This also includes `@types/jquery` and `@types/jqueryui`

Reply:
jQuery is still used in at least 1 place, should we refactor it?

https://github.com/findologic/plugin-plentymarkets-ceres-api/blob/PLENTY-444-migrating_remaining_components/resources/js/src/app/components/itemList/filter/ItemRangeSlider.vue#L163

Reply:
Seems like it is using the jquery of the shop, and not the package, as the package would need to be imported somewhere.

Reply:
It is also used in tests
![image](https://user-images.githubusercontent.com/41129367/223722199-1024a916-0054-4c8c-b512-852feb70a6a1.png)

Reply:
Ok, then let's leave it for now